ndbench-core/src/main/java/com/netflix/ndbench/core/defaultimpl/NdBenchGuiceModule.java

```diff
@@ -39,8 +40,14 @@
     protected void configure()
     {
         bind(NdBenchMonitor.class).to(FakeMonitor.class);
-        //bind(IClusterDiscovery.class).to(AWSLocalClusterDiscovery.class);
-        bind(IClusterDiscovery.class).to(LocalClusterDiscovery.class);
+        String discoveryEnv = System.getenv(NdBenchConstants.DISCOVERY_ENV);
```

Comment:
It seems we are making the binding based on some constants rather providing one by default. What is the logic behind this change? (I'm not against it just trying to understand)